x-pack/plugins/siem/public/components/timeline/body/renderers/empty_column_renderer.tsx

```diff
@@ -44,9 +44,10 @@ export const emptyColumnRenderer: ColumnRenderer = {
         queryMatch: {
           field: field.id,
           value: parseQueryValue(null),
-          operator: IS_OPERATOR,
+          displayValue: getEmptyValue(),
+          operator: EXISTS_OPERATOR,
```

Comment:
Really nice fix -- this works great well with the new `Edit Filter` UI. Nice job @XavierM :)